### PR TITLE
Add `raw_logits` in GraphActionCategorical to() method

### PR DESCRIPTION
**Pull request by bengioe**
`raw_logits` were not sent to the device, causing downstream errors depending on when the raw logits are used to compute the masked logits.

### CODE REVIEW

**Comment by bengioe** [comment]
~Not sure why the Mac build fails, seems to be some bad config on github's side. Either way this is a minor fix and passes on Linux.~ Fixed by #132

**Comment by bengioe** [comment]
@julienroyd things seem to work with this fix but I haven't extensively tested. Can you think of other places where we should check for a similar error?

**Comment by julienroyd** [comment]
> @julienroyd things seem to work with this fix but I haven't extensively tested. Can you think of other places where we should check for a similar error?

No, I think it should be good, _masked_logits were already being sent to device so I think raw_logits were the only ones that were left behind.

**Review by julienroyd — APPROVED** [review]
